spec/components/candidate_interface/restructured_work_history_break_placeholder_component_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::RestructuredWorkHistoryBreakPlaceholderComponent do
+  let(:break_period) { double }
+
+  before do
+    allow(break_period).to receive(:length).and_return(3)
+    allow(break_period).to receive(:start_date).and_return(Date.new(2020, 1, 1))
+    allow(break_period).to receive(:end_date).and_return(Date.new(2020, 5, 1))
+  end
+
+  it 'renders the component with a link to explain break' do
+    result = render_inline(CandidateInterface::RestructuredWorkHistoryBreakPlaceholderComponent.new(break_period: break_period))
+
+    expect(result.css('a').last.text).to eq('add a reason for this break')
+  end
+
+  it 'renders the component with a link to add another job' do
+    result = render_inline(CandidateInterface::RestructuredWorkHistoryBreakPlaceholderComponent.new(break_period: break_period))
+
+    expect(result.css('a').first.text).to eq('Add another job')
+  end
+
+  it 'renders the component with the break length' do
+    result = render_inline(CandidateInterface::RestructuredWorkHistoryBreakPlaceholderComponent.new(break_period: break_period))
+
+    expect(result.text).to include('You have a break in your work history (3 months)')
+  end
```

Comment:
I think we need to add another test to here that checks this goes higher than 5 years and change the logic in the service which calculates breaks to have this functionality.

At the moment it's capped at 5 

![Uploading image.png…]()

Reply:
@davidgisbey yep, I think we need to change the logic so that the gap placeholder appears between any two items, rather than only in the past 5 years. I can't remember if the current application also prompts for a gap before the first item or not - but if it does we could probably remove that.